src/components/panel/panel.spec.js

```diff
@@ -1426,46 +1423,45 @@ describe('$mdPanel', function() {
    * @param {!Object=} opt_config
    */
   function openPanel(opt_config) {
+    // TODO(ErinCoughlan): Investigate why panelRef.open() doesn't return
+    // panelRef.
+    var openPromise;
     if (opt_config) {
-      panelRef = $mdPanel.open(opt_config);
+      openPromise = $mdPanel.open(opt_config);
     } else if (panelRef) {
-      panelRef.open();
+      openPromise = panelRef.open();
     } else {
-      panelRef = $mdPanel.open();
+      openPromise = $mdPanel.open();
```

Comment:
(nit) This could be just two conditions:
```
if (panelRef) {
  openPromise = panelRef.open();
}
else {
  openPromise = $mdPanel.open(opt_config);
}
```